### ISSUE TITLE
Changes in files causes an attribute error for python 3.5

### ISSUE THREAD

**Issue by sarahttan** [borntyping/spotter#10]
I might have been screwing this one up, but, when I try to run the sample > `spotter` command in the spotter repo and a file changes, I get the following attribute error:
`AttributeError: 'str' object has no attribute 'decode'`
Would you happen to have any ideas on what to do?  I've built spotter with the python3.5 distribution.

**Comment by borntyping** [comment]
I don't think this has ever been tested on Python 3, sorry (I've not done any work on it for many years). I'd accept any PRs that updated this to work on Python 3 even if they broke backwards compatibility though.
